osu.Game/Rulesets/UI/DrawableRuleset.cs

```diff
@@ -98,6 +98,14 @@ internal override bool FrameStablePlayback
 
         private DrawableRulesetDependencies dependencies;
 
+        /// <summary>
+        /// An audio container which can be used to apply adjustments to playfield content.
+        /// </summary>
+        /// <remarks>
+        /// Does not affect <see cref="Overlays"/>.
+        /// </remarks>
+        public AudioContainer AudioContainer { get; private set; }
```

Comment:
Naming this just `Audio` reads better, or `PlayfieldAudio` to be explicit.

Reply:
If we're going down that route, it should only be exposed as the `IAudioAdjustment` interface, not the container.

Reply:
Have made this change.